deploy.sh

```diff
@@ -0,0 +1,3 @@
+./gradlew clean assembleProductionDebug
+
+curl -F "file=@app/build/outputs/apk/app-production-debug.apk" -F "token=${DEPLOY_GATE_API_KEY}" -F "message=https://circleci.com/gh/konifar/droidkaigi2016/${CIRCLE_BUILD_NUM}" -F "distribution_key=0d34ff6b3ccedf2a76771d9b6002c55b4f0aac1f" -F "release_note=https://circleci.com/gh/konifar/droidkaigi2016/${CIRCLE_BUILD_NUM}" https://deploygate.com/api/users/konifar/apps
```

Comment:
After create release keystore, change `app-production-debug.apk` to `app-production-release.apk`.